generator.go

```diff
@@ -144,3 +149,88 @@ func sign(message, secret string) string {
 	h.Write([]byte(message))
 	return encode(h.Sum(nil))
 }
+
+func GenerateCustomToken(uid string, developerClaims *jws.Claims, clientEmail string, privateKeyString string) (string, error) {
```

Comment:
Could you make the developerClaims a `map[string]interface{}` so that we are not tied to the `github.com/SermoDigital/jose/jws` package?